lib/include/aws_iot_defender.h

```diff
@@ -101,39 +107,39 @@ typedef struct AwsIotDefenderStartInfo
     AwsIotDefender_SetMetrics(                          \
         metricsArray, sizeof( metricsArray ) / sizeof( AwsIotDefenderMetric_t ) )
 
- /**
-  * Set an array of metrics.
-  */
-AwsIotDefenderError_t AwsIotDefender_SetMetrics(const AwsIotDefenderMetric_t * const pMetricsArray,
-	size_t metricsCount);
+/**
+ * Set an array of metrics.
+ */
+AwsIotDefenderError_t AwsIotDefender_SetMetrics( const AwsIotDefenderMetric_t * const pMetricsArray,
+                                                 size_t metricsCount );
 
 /**
  * Start the defender */
-AwsIotDefenderError_t AwsIotDefender_Start(AwsIotDefenderStartInfo_t * pStartInfo);
+AwsIotDefenderError_t AwsIotDefender_Start( AwsIotDefenderStartInfo_t * pStartInfo );
 
 /**
  * Stop the defender
  */
-AwsIotDefenderError_t AwsIotDefender_Stop(void);
+AwsIotDefenderError_t AwsIotDefender_Stop( void );
 
 /**
  * Set period in seconds.
  */
-AwsIotDefenderError_t AwsIotDefender_SetPeriod(uint64_t periodSeconds);
+AwsIotDefenderError_t AwsIotDefender_SetPeriod( uint64_t periodSeconds );
 
 /**
  * Get period in seconds.
  */
-uint32_t AwsIotDefender_GetPeriod(void);
+uint32_t AwsIotDefender_GetPeriod( void );
```

Comment:
AwsIotDefender_SetPeriod() takes a uint64_t as the period but AwsIotDefender_GetPeriod() returns uint32_t. Can we make it consistent ?

Reply:
It is my mistake. I will fix it